CHANGELOG.rst

```diff
@@ -14,6 +23,8 @@ Individual Components
 ~~~~~~~~~~~~~~~~~~~~~
 
 - `renku-data-services 0.15.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.15.0>`
+- `renku-notebooks 1.25.2 <https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/1.25.2>`_
```

Comment:
Notebooks is mentioned here but not updated in the values. Is this correct?

Reply:
Other PRs bumping the notebooks version got merged to the release branch in the meanwhile, but they don't mention it in the individual components list :shrug: